extensions/jt400/deployment/src/main/java/org/apache/camel/quarkus/component/jt400/deployment/Jt400Processor.java

```diff
@@ -40,4 +49,30 @@ List<RuntimeInitializedClassBuildItem> runtimeInitializedClasses() {
         items.add(new RuntimeInitializedClassBuildItem("com.ibm.as400.access.CredentialVault"));
         return items;
     }
+
+    @BuildStep
+    void reflectiveClasses(BuildProducer<ReflectiveClassBuildItem> reflectiveClassesProducer,
+            CombinedIndexBuildItem combinedIndex) {
+        IndexView index = combinedIndex.getIndex();
+
+        reflectiveClassesProducer.produce(ReflectiveClassBuildItem.builder(NLSImplNative.class).build());
+
+        index.getKnownClasses().stream()
+                .filter(c -> c.name().toString().matches("com.ibm.as400.access.*Remote"))
```

Comment:
Nitpick - might be better for performance to use a `Pattern` constant instead of doing regex compilation in loops that are somewhat unbounded.

Reply:
that's right, I'll take care of that